plugins/modules/foreman_user.py

```diff
@@ -394,7 +394,7 @@ def main():
     module.connect()
 
     search = 'login="{}"'.format(entity_dict['name'])
-    entity = module.find_resource('users', search, failsafe=True)
+    entity = module.find_resource('users', search, failsafe=True, thin=False)
```

Comment:
why not the default?

Reply:
The current recorded delete task did not use thin=True.

Reply:
same comment about comments as above then :)